README.md

```diff
@@ -1,7 +1,8 @@
 # Blue Manual
 
-This is the source code for the Blue Manual site. The project uses
-[MkDocs](https://mkdocs.org) and [Material for MkDocs](https://squidfunk.github.io/mkdocs-material) for building the manual.  To work with the manual
+This is not the source code for the Blue Manual site. These are my efforts to improve the manual. Official source code can be found here: https://github.com/kunstmusik/blue-manual
```

Comment:
This is part isn't necessary as it will be merged into the original, could you add commit to this PR that undoes this change?  Otherwise, I can just merge this PR and undo it here, just let me know.